lib/settings.py

```diff
@@ -13,21 +13,22 @@
 import lib.output
 import lib.banner
 
+CUR_DIR = "{}".format(os.getcwd())
```

Comment:
One variable, or ten more function calls.

Reply:
I mean it's not much different, but I see where you're coming from